tests/common/bats/update-ips.bash

```diff
@@ -134,7 +120,9 @@ update_ips.mock_swift() {
   update_ips.mock_minimal
 
   # GET /auth/tokens
-  mock_set_output "${mock_curl}" '{"token":{"catalog":[{"type": "object-store", "name": "swift", "endpoints": [{"interface":"public", "region": "swift-region", "url": "https://swift.foo.dev-ck8s.com:91011"}]}]}}' 1
+  mock_set_output "${mock_curl}" 'x-subject-token: 123456789
+
+  {"token":{"catalog":[{"type": "object-store", "name": "swift", "endpoints": [{"interface":"public", "region": "swift-region", "url": "https://swift.foo.dev-ck8s.com:91011"}]}]}}' 1
```

Comment:
I think it's good if we make this a bit more realistic so that we don't assume it works only because it passes a test with one header in it.

Something like this?

```suggestion
  mock_set_output "${mock_curl}" 'HTTP/2 200
date: Wed, 09 Oct 2024 14:14:40 GMT
expires: -1
cache-control: private, max-age=0
content-type: text/html; charset=ISO-8859-1
x-subject-token: 123456789
accept-ranges: none
vary: Accept-Encoding

  {"token":{"catalog":[{"type": "object-store", "name": "swift", "endpoints": [{"interface":"public", "region": "swift-region", "url": "https://swift.foo.dev-ck8s.com:91011"}]}]}}' 1
```

You could change it to headers that resemble the actual output better though.

Reply:
Good point, I will add them :+1: